app/queues/notifications/expiration-notification.server.ts

```diff
@@ -0,0 +1,189 @@
+// Third party modules
+import { Worker, Queue } from 'bullmq';
+
+// Internal modules
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
+
+// Types
+import type { ExpirationStatusPayload } from './record-cert-expiration-data.interface';
```

Comment:
Because this type info is only used in this file, move the types into this file.